lib/lobbyist/v2/data_warehouse_job.rb

```diff
@@ -0,0 +1,14 @@
+module Lobbyist
+  module V2
+    class DataWarehouseJob < Lobbyist::V2::Base
+      attr_accessor :id, :company_id, :status,
+                    :params, :data_filename, :result_filename,
+                    :job_type, :completed_at
+
+      def self.update(id, params)
+        create_from_response(
+            put("/v2/data_warehouse_job/#{id}.json", params))
```

Comment:
Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.